docs/website/docs/guides/deployment-configurations/cpu.md

```diff
@@ -104,12 +104,53 @@ drivers:
 ```console hl_lines="4 5"
 $ iree-run-module --list_drivers
 
-        cuda: CUDA (dynamic)
-  local-sync: Local execution using a lightweight inline synchronous queue
-  local-task: Local execution using the IREE multithreading task system
-      vulkan: Vulkan 1.x (dynamic)
+            cuda: NVIDIA CUDA HAL driver (via dylib)
+             hip: HIP HAL driver (via dylib)
+      local-sync: Local execution using a lightweight inline synchronous queue
+      local-task: Local execution using the IREE multithreading task system
+          vulkan: Vulkan 1.x (dynamic)
 ```
 
+You can download the runtime from a release or build from source.
+
+#### :octicons-package-16: Download the runtime from a release
```

Comment:
The other "deployment configuration" pages also have a "Get the IREE runtime" section, like https://iree.dev/guides/deployment-configurations/gpu-vulkan/#get-the-iree-runtime. I think we should update them all together.